tests/ci/run_check.py

```diff
@@ -176,29 +161,46 @@ def main():
         )
         sys.exit(1)
 
-    set_mergeable_check(commit, "skipped")
-    ci_report_url = create_ci_report(pr_info, [])
-    if not can_run:
-        print("::notice ::Cannot run")
+    if FEATURE_LABEL in pr_info.labels and not pr_info.has_changes_in_documentation():
+        print(
+            f"The '{FEATURE_LABEL}' in the labels, "
+            "but there's no changed documentation"
+        )
+        PR_CHECK = "PR Check"
         post_commit_status(
             commit,
-            labels_state,
-            ci_report_url,
-            description,
-            CI_STATUS_NAME,
+            FAILURE,
+            "",
+            f"expect adding docs for {FEATURE_LABEL}",
+            PR_CHECK,
             pr_info,
         )
-        sys.exit(1)
+        update_mergeable_check(commit, pr_info, PR_CHECK)
```

Comment:
when we use `at_exit`, we have a gap between the `port_commit_status` and the following `update_mergeable_check`

Thanks to it, the probability of getting the correct value is higher. Let's use it here too